### PR TITLE
remove redundant parentheses

### PR DESCRIPTION
**Pull request by jinglina**
remove redundant parentheses

### CODE REVIEW

**Review by skitt — APPROVED** [review]

**Comment by jinglina** [comment]
ready-to-test

**Review by tpantelis — APPROVED** [review]

File: pkg/cableengine/healthchecker/statistics.go [hunk 1]
```diff
@@ -64,7 +64,7 @@ func (s *statistics) update(rtt uint64) {
 		s.sum += rtt
 		oldMean := s.mean
 		s.mean = s.sum / (s.index + 1)
-		s.sqrDiff += uint64(((int64(rtt - oldMean)) * int64((rtt - s.mean))))
+		s.sqrDiff += uint64((int64(rtt - oldMean)) * int64(rtt - s.mean))
```
Comment (skitt) [inline pkg/cableengine/healthchecker/statistics.go:67]: To pass our linting, the spaces around the `-` operators need to be removed; in addition, there’s another set of parentheses which can be removed:

```suggestion
		s.sqrDiff += uint64(int64(rtt-oldMean) * int64(rtt-s.mean))
```

**Review by skitt — CHANGES_REQUESTED** [review]

File: pkg/cableengine/healthchecker/statistics.go (hunk 1 above)
Comment (jinglina) [reply pkg/cableengine/healthchecker/statistics.go:67]: Ok, thanks

**Comment by dfarrell07** [comment]
Thanks for bringing this to our attention @jinglina! I think @skitt has a more generic fix here: https://github.com/submariner-io/submariner/pull/1770

**Comment by skitt** [comment]
> Thanks for bringing this to our attention @jinglina! I think @skitt has a more generic fix here: #1770

@jinglina if you would like to fix your PR, I’d be very happy to rebase mine on top of yours, so you get credit. I don’t want to replace your PR with mine!

**Comment by dfarrell07** [comment]
That sounds great, +1 to letting the commit history reflect the development history (and giving credit!).

Just need to fix the Go linting here.

**Comment by jinglina** [comment]
I will abandon this PR here. Your solution is obviously better. Thanks.

**Pull request closed without merging**